### ISSUE TITLE
Commands do not work + missing UI

### ISSUE THREAD

**Issue by strnglp** [elanthia-online/illthorn#249]
Running `electron-22` branch from the tip or the last tagged release the client seems not quite in working order.
1. ? icon isn't there in the bottom right
2. : prefix commands do not work

I tried to build `master` but it fails downloading an older electron.

**Comment by ondreian** [comment]
Which `:` commands did you try?

**Comment by strnglp** [comment]
All of them. I am using the electron-22 branch and I noticed :config and :theme doesn't appear in the code anymore? It used to be in a folder called vimish in the master branch. I had assumed you refactored significantly and haven't gotten around to reintroducing the concept.

Note I also tried the release zips and had the same result.

**Comment by strnglp** [comment]
Is this still actively developed? Does it work for you?

**Comment by ondreian** [comment]
I am working on some things on my localhost, I use it as my daily driver for an FE, but I pruned a lot of stuff back because the scope had creeped a bit in several places.

**Comment by strnglp** [comment]
Understood, if you do wind up pushing the changes up I'd be happy to test and attempt bug fixes. I think what you've built is great and gives players options. For instance I want to run this in linux and I don't know of any other options.

**Comment by ondreian** [comment]
I will see about syncing up my changes tonight or tomorrow morning. 🙂

**Comment by litch** [comment]
Gentle nudge on this thanks!  Looks like a pretty intense project to be sure but something it may be fun to see more progress on.

**Comment by ondreian** [comment]
If you are using modern lich, it will autodetect the session and connect if active.

**Comment by litch** [comment]
Right.  Connects just fine, but the previously-documented `:commands` don't work, and none of the theme switching seems to work.  I've been able to go in and monkey with some of the TS and CSS and have it looking & feeling nicer.  I kind of got the impression that you did a bunch of work to bring it to compliance with electron-22, but wound up with a massive amount of cruft around, but the "happy path" of it generally works for you, which is cool.

**Comment by ondreian** [comment]
yes, the added overhead of refactoring all of the themes was not something
I am interested in right now, when i work on it i try to add novel
features.  There is a significant need for several of the commands to be
refactored to the new electron security policies as well.

**Comment by litch** [comment]
Yes that makes sense.  You mentioned using it as your daily driver, is that a branch that you've just got local that you're using actively?  I did a fair amount of rework on the UI and it feels a lot nicer *to me*, but would be interested in actual feature work that may be interesting (more nimble highlights?)

**Comment by ondreian** [comment]
Fairly sure it's the current electron-22 branch right now, what do you mean
by nimble? I'm not opposed to PRs :smile:
